frontend/packages/ceph-storage-plugin/src/constants/queries.ts

```diff
@@ -87,6 +89,13 @@ export const CAPACITY_BREAKDOWN_QUERIES = {
     'max(ceph_pool_max_avail * on (pool_id) group_left(name)ceph_pool_metadata{name=~"(.*file.*)|(.*block.*)"})',
 };
 
+export const POOL_STORAGE_EFFICIENCY_QUERIES = {
```

Comment:
FYI @openshift/openshift-team-monitoring

Reply:
nice, thank you!

Reply:
that is an incarnation of centralizing queries, correct?

Reply:
Ah no. The centralizing PR is https://github.com/openshift/console/pull/6128